### PR TITLE
Update DG with the suggested format for summarise command and pie chart

### PR DESCRIPTION
**Pull request by jiale-c**
Standardised format of command section with the rest of the features.

This PR is standalone from other PRs. This PR requires others to vet and approve.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -255,20 +255,42 @@ Following this, `LogicManager` will call the `execute()` method of the `AddComma
...
-In order for this feature to be unique and not overlap what the List feature has to offer, summarise helps to calculate how many
-students who are covid positive in each block of the hall, alongside those who are negative and on health risk notice.
-This helps the hall masters determine if there is a spread of virus in any particular block.
 
-Tracey will then calculate those that are positive and which faculty they come from. This is helpful to determine if the superspreader
-comes from the faculty building itself. The hall masters and leaders can be more certain on their follow up actions to keep
-their hall safe.
+#### What is the summarise feature
+
+The summarise feature allow users to visualise the statistics of students in the Hall by their covid status and their faculty/block.
```
Comment (tanchinkiat99) [inline docs/DeveloperGuide.md:263]: Very minor grammatical change 'allow' should be 'allows'.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -255,20 +255,42 @@ Following this, `LogicManager` will call the `execute()` method of the `AddComma
...
 
 **Path Execution of Summarise Feature Activity Diagram is shown below:**
 ![SummariseFeatureActivityDiagram](images/SummariseFeatureActivityDiagram.png)
 
+There are three possible execution paths for this command.
+1. User inputs `summarise` command with students' records stored in Tracey. After the Pie Chart Window opens, the user can scroll through the window to see the pie charts for each block followed by the bar chart categorised by faculty.
+2. User inputs `summarise` command with no students' records stored in Tracey. Tracey will just respond that there is no students to summarise. The Pie Chart Window will not open.
+3. User inputs `summarise` command with additional parameters. Tracey will throw a **ParseException** to indicate that the format of the summarise input is wrong.
+
+The sequence diagram below shows the interactions between objects during the execution of a `summarise` command.
```
Comment (tanchinkiat99) [inline docs/DeveloperGuide.md:284]: I think this line should come after the class diagram.

**Review by cwq2326 — APPROVED** [review]
LGTM

**Review by tanchinkiat99 — APPROVED** [review]
LGTM, just a few minor changes.

**Pull request merged**